### ISSUE TITLE
Muse - Starlight (Ultimate CON version) is not loaded at all

### ISSUE THREAD

**Issue by HideriFN** [C0Assassin/rb4-to-rb2#10]
I already try this song on RB3 (PS3, convert this song using Onyx) and YARG but this song is not loaded at all. Can you guys fix it? Thanks.

**Comment by C0Assassin** [comment]
Ill look into it.

**Comment by afestini** [comment]
The song_tonality entries have a comment that is missing the ";", resulting in a corrupt file.

If you can edit the files yourself, there are two instances of this line:
`(song_tonality 0) - Only set this for pro guitar upgrades and if you know what you are doing`

Add the missing ; to fix the comment, or better, replace the "-" with ";", which can be done in a hex editor without requiring any fancy tools (I first used c3con and it promptly corrupted the file with an invalid block number for the volume descriptor)

`(song_tonality 0) ; Only set this for pro guitar upgrades and if you know what you are doing`

**Comment by C0Assassin** [comment]
Is this the only song like this? Thats very strange that it ended up coming up with this error. Ill check it out as soon as I can, and push a fix.

**Comment by afestini** [comment]
I can't be 100% sure just from the numbers, since some files contain multiple versions with #ifdefs, so it's not an exact 1:1 mapping of files vs. songs. It was the only one that caused noticeable issues.

The only other thing I ran into was "Knights of Cydonia" having a bass volume of -96 (way outside the -10 to +10 I found mentioned somewhere). Unclear whether that's also a typo. In my case it sounds extremely wrong, but that's because I trusted the -10/+10 and don't clamp.

Is it known whether the gain adjustments are linear or supposed to be in db? Though in either case, I would expect -96 to be practically muted.
